app/_hub/kong-inc/request-validator/index.md

```diff
@@ -48,24 +49,27 @@ params:
       required: true
       default: "kong"
       value_in_examples:
+      datatype: string
       description: |
-        What validator to use. Supported values are `kong` (default) for using Kong's own schema
+        Which validator to use. Supported values are `kong` (default) for using Kong's own schema
         validator, or `draft4` for using a JSON Schema Draft 4-compliant validator.
 
     - name: parameter_schema
       required: false
```

Comment:
required = false is not quite correct, see my other comment

Reply:
changed to semi since there is a dependency